### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/03/day2.html?fbclid=IwAR3EBRxU_yS7tUY6EgIYxrRRFRjjzauWxxSBnN_h2Za0JEma3Q5yLtXFbYY_aem_AYy_KUKvpqh9gKFL7fCtWDjJZuRu58_IXBOTgWYRC9FFgaMAMqVj_7qcXggMbB6mKrA

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#11] [Comments]
# https://mnfienen.github.io/talulat/2023/08/03/day2.html?fbclid=IwAR3EBRxU_yS7tUY6EgIYxrRRFRjjzauWxxSBnN_h2Za0JEma3Q5yLtXFbYY_aem_AYy_KUKvpqh9gKFL7fCtWDjJZuRu58_IXBOTgWYRC9FFgaMAMqVj_7qcXggMbB6mKrA

Day 2 - An early rest day and an opportunity to hang out with friends and do some logistics miles today: 12 total miles: 76 km today: 20 total km: 142

https://mnfienen.github.io/talulat/2023/08/03/day2.html?fbclid=IwAR3EBRxU_yS7tUY6EgIYxrRRFRjjzauWxxSBnN_h2Za0JEma3Q5yLtXFbYY_aem_AYy_KUKvpqh9gKFL7fCtWDjJZuRu58_IXBOTgWYRC9FFgaMAMqVj_7qcXggMbB6mKrA

**Comment by SmittysGitHub** [comment]
Congrats on getting the squeak fixed!  Good luck with the rest!
